ts/opengroup/opengroupV2/OpenGroupAPIV2.ts

```diff
@@ -264,16 +264,18 @@ export const postMessage = async (
 
 export const banUser = async (
   userToBan: PubKey,
-  roomInfos: OpenGroupRequestCommonType
+  roomInfos: OpenGroupRequestCommonType,
+  deleteAllMessages: boolean
 ): Promise<boolean> => {
   const queryParams = { public_key: userToBan.key };
+  const endPoint = deleteAllMessages ? 'ban_and_delete_all' : 'block_list';
   const request: OpenGroupV2Request = {
     method: 'POST',
     room: roomInfos.roomId,
     server: roomInfos.serverUrl,
     isAuthRequired: true,
     queryParams,
-    endpoint: 'block_list',
+    endpoint: endPoint,
```

Comment:
same issue, just make endPoint a with the shift P
so you can just use `endpoint,` instead of `endpoint: endPoint,`